stylesheets/_modules.scss

```diff
@@ -783,8 +785,8 @@
 .module-message__generic-attachment__text {
   flex-grow: 1;
   margin-left: 8px;
-  // The width of the icon plus our 8px margin
-  max-width: calc(100% - 37px);
+  // The width of the icon plus our 8px margin plus 1px leeway
+  max-width: calc(100% - 36px);
```

Comment:
Could you describe what this fixes? I don't see any visual changes when applying it and the current comment says 37 is indeed the sum of the margin and the icon width.

Reply:
I found that this change has an effect when you are zoomed in at least one level more than the default zoom (for instance by pressing Ctrl+=). Then, without this change, file names get truncated no matter what, even if they are short. I came up with this fix after inspecting the html element and finding that its size had fractional values.

Without the change:
![overflow_1](https://user-images.githubusercontent.com/4296166/114078095-829fb680-9876-11eb-87c7-f67bff0f2613.png)
With the change:
![overflow_2](https://user-images.githubusercontent.com/4296166/114078117-87646a80-9876-11eb-8d07-f93cc05ab3eb.png)
Screenshot showing the element size has fractional values:
![overflow_3](https://user-images.githubusercontent.com/4296166/114080050-e75c1080-9878-11eb-8075-fe5c896b9558.png)

Reply:
Oh, tricky! Alright. LGTM!